check-version-bump.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
```

Comment:
Good idea! The makefile should reflect the CI config… or is it the other way around? :wink:

I don't think that's blocking for this PR.